shell/src/main/java/tachyon/shell/TfsShell.java

```diff
@@ -730,6 +735,10 @@ public int getCapacityBytes() {
   public int pin(TachyonURI path) {
     try {
       TachyonFile fd = mTfs.open(path);
+      if (fd == null) {
```

Comment:
I need to `return -1` in this kind of blocks, if it's abstracted to a function, extra effort is needed to check the return value of the function, no as straightforward as current code.